### PR TITLE
Fix article selected text context menu "Edit" action not selecting text in the editor

### PR DESCRIPTION
**Pull request by joewalsh**
Update (unfortunately still necessary) method swizzling for iOS 13

https://phabricator.wikimedia.org/T240335

### CODE REVIEW

File: Wikipedia/Code/WKWebViewWithSettableInputViews.swift [hunk 1]
```diff
@@ -89,24 +89,53 @@ class WKWebViewWithSettableInputViews: WKWebView {
...
+        let sel_13_0: Selector = sel_getUid("_elementDidFocus:userIsInteracting:blurPreviousNode:activityStateChanges:userObject:")
+
+        if let method = class_getInstanceMethod(WKContentView, sel_11_3) {
+            let originalImp: IMP = method_getImplementation(method)
+            let original: NewClosureType = unsafeBitCast(originalImp, to: NewClosureType.self)
+            let block : @convention(block) (Any, UnsafeRawPointer, Bool, Bool, Bool, Any?) -> Void = { (me, arg0, arg1, arg2, arg3, arg4) in
+                original(me, sel_11_3, arg0, !value, arg2, arg3, arg4)
+            }
+            let imp: IMP = imp_implementationWithBlock(block)
+            method_setImplementation(method, imp)
```
Comment (montehurd) [inline Wikipedia/Code/WKWebViewWithSettableInputViews.swift]: would adding a `return` after these `method_setImplementation` calls and reversing the order to 13.0, 12.2, 11.3 be a bit more efficient?

File: Wikipedia/Code/WKWebViewWithSettableInputViews.swift [hunk 2]
```diff
@@ -89,24 +89,53 @@ class WKWebViewWithSettableInputViews: WKWebView {
...
         guard let WKContentView: AnyClass = NSClassFromString("WKContentView") else {
-            DDLogError("Could not get class")
+            print("keyboardDisplayRequiresUserAction extension: Cannot find the WKContentView class")
             return
         }
-        let sel = sel_getUid("_startAssistingNode:userIsInteracting:blurPreviousNode:changingActivityState:userObject:")
-        guard let method = class_getInstanceMethod(WKContentView, sel) else {
-            DDLogError("Could not get method")
-            return
+        // For iOS 11.3, *
```
Comment (montehurd) [inline Wikipedia/Code/WKWebViewWithSettableInputViews.swift]: good catch on the methods signatures!

File: Wikipedia/Code/WKWebViewWithSettableInputViews.swift (hunk 1 above)
Comment (joewalsh) [reply Wikipedia/Code/WKWebViewWithSettableInputViews.swift]: It's possible multiple are implemented and return-ing would shortcut that, but I haven't checked. Given this is only called once and there doesn't seem to be any performance degradation, I'm ok with leaving it as is

File: Wikipedia/Code/WKWebViewWithSettableInputViews.swift (hunk 2 above)
Comment (joewalsh) [reply Wikipedia/Code/WKWebViewWithSettableInputViews.swift]: I can't take credit, it's a stack overflow answer linked above

Comment (montehurd) [reply Wikipedia/Code/WKWebViewWithSettableInputViews.swift]: i did notice this https://stackoverflow.com/a/55344531 example DRY's the swift up a bit - might be nice

Comment (montehurd) [reply Wikipedia/Code/WKWebViewWithSettableInputViews.swift]: but i don't feel strongly about it :)

**Pull request merged**